tests/conftest.py

```diff
@@ -445,6 +475,63 @@ def pytest_collection_modifyitems(session, config, items):
                     f" UI is not supported on {ocsci_config.ENV_DATA['platform'].lower()}"
                 )
                 items.remove(item)
+    # If multicluster upgrade scenario
+    if ocsci_config.multicluster and ocsci_config.UPGRADE.get("upgrade", False):
+        for item in items:
+            if (
+                list(
+                    set([i.name for i in item.iter_markers()]).intersection(
+                        (MultiClusterUpgradeParametrize.MULTICLUSTER_UPGRADE_MARKERS)
+                    )
+                )
+                and ocsci_config.multicluster
```

Comment:
this part of the condition `and ocsci_config.multicluster` is probably not necessary here as it is also part of the condition on line 479:
```python
if ocsci_config.multicluster and ocsci_config.UPGRADE.get("upgrade", False):
```